### PR TITLE
[KIECLOUD-93] .gitignore changes to fix vendor accuracy

### PR DESCRIPTION
**Pull request by tchughesiv**
Signed-off-by: tchughesiv <tchughesiv@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -71,8 +71,8 @@ anaconda-mode/
 Session.vim
 # temporary
 .netrwhist
-# auto-generated tag files
-tags
```
Comment (ruromero) [inline .gitignore]: Adding this at the end of the file might be better
```
!vendor/**
```

**Comment by tchughesiv** [comment]
necessary changes applied to https://github.com/kiegroup/kie-cloud-operator/pull/49 instead

**Pull request closed without merging**